ui/apps/platform/src/Components/CompoundSearchFilter/components/CompoundSearchFilter.cy.jsx

```diff
@@ -70,7 +70,7 @@ describe(Cypress.spec.relative, () => {
     it('should display nothing in the entity selector', () => {
         const config = {};
 
-        setup(config);
+        setup(config, () => {});
```

Comment:
I was going to test out the `onSearch`, but decided maybe it would be best to test that out fully through E2E tests

Reply:
I think it is worth adding as a follow up if we have time, given how widespread these components will be in the app.

I've had problems in the past testing `onSearch` functionality due to how unpredictable the search results can be in an e2e environment, at least when testing on CVE based data.